CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.1.0] - 2019-06-04
+
+### Added
+
+- Add push-to-app-catalog job.
+
+[Unreleased]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.1.0...HEAD
+[0.1.0]: https://github.com/olivierlacan/keep-a-changelog/releases/tag/v0.1.0
```

Comment:
Wait, why do we have this url there ?

Reply:
I pushed the tag manually for the commit with job promoting to prod. I'll describe the development process here in the later PR.

Reply:
Ah. And this should be `giantswarm/architect-orb`... Changing.

Reply:
Good catch, thanks!